merp_outgoing_routing/models/res_company.py

```diff
@@ -9,15 +9,24 @@ class Company(models.Model):
 
     outgoing_routing_strategy = fields.Selection(
         [
-            ('name', 'Sort by source locations in alphabetical order'),
-            ('removal_prio', 'Sort by location removal strategy priority'),
-            ('product', 'Sort by product name'),
+            # path should be valid for both stock pickings and quants
+            ('location_id.removal_prio', 'Location removal priority'),
+            ('location_id.name', 'Location name'),
+            ('product_id.name', 'Product name'),
         ],
-        string='Routing Strategy', default='name')
+        string='Routing Strategy', default='location_id.name')
 
     outgoing_routing_order = fields.Selection(
         [
             ('0', 'Ascending (A-Z)'),
             ('1', 'Descending (Z-A)'),
         ],
         string='Routing Order', default='0')
+
+    stock_reservation_strategy = fields.Selection(
```

Comment:
I think module conflicts with this changes . Maybe they should be removed? https://github.com/ventor-tech/merp/pull/71/files

Also another important point: inside location still there should be applied default pull rule (FIFO, LIFO, Expired First). Even if custom strategy applied. So logic is the following :
1) if "Default" is selected, than apply only standard rules
2) if other option is selected, than apply new rules. If found multiple similar options , than apply standard  rules

Reply:
it works (this way)